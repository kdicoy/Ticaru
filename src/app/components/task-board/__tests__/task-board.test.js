import React from 'react';
import { shallow } from 'enzyme';
import { TaskBoard } from '../index';

const initialState = {
  weeklyBoard: {
    Monday: [],
    Tuesday: [],
    Wednesday: [],
    Thursday: [],
    Friday: [],
    Saturday: [],
    Sunday: [],
  },
};
describe('WeeklyBoard', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<TaskBoard debug {...initialState} />);

    expect(component).toMatchSnapshot();
  });
});
