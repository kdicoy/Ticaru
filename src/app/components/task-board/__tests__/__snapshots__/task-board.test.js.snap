// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeeklyBoard should render correctly in "debug" mode 1`] = `
<Fragment>
  <DragDropContext
    onDragEnd={[Function]}
  >
    <div
      style={
        Object {
          "padding": 20,
        }
      }
    >
      <Connect(Droppable)
        direction="vertical"
        droppableId="Monday"
        ignoreContainerClipping={false}
        isDropDisabled={false}
        type="DEFAULT"
      >
        <Component />
      </Connect(Droppable)>
      <Connect(Droppable)
        direction="vertical"
        droppableId="Tuesday"
        ignoreContainerClipping={false}
        isDropDisabled={false}
        type="DEFAULT"
      >
        <Component />
      </Connect(Droppable)>
      <Connect(Droppable)
        direction="vertical"
        droppableId="Wednesday"
        ignoreContainerClipping={false}
        isDropDisabled={false}
        type="DEFAULT"
      >
        <Component />
      </Connect(Droppable)>
      <Connect(Droppable)
        direction="vertical"
        droppableId="Thursday"
        ignoreContainerClipping={false}
        isDropDisabled={false}
        type="DEFAULT"
      >
        <Component />
      </Connect(Droppable)>
      <Connect(Droppable)
        direction="vertical"
        droppableId="Friday"
        ignoreContainerClipping={false}
        isDropDisabled={false}
        type="DEFAULT"
      >
        <Component />
      </Connect(Droppable)>
      <Connect(Droppable)
        direction="vertical"
        droppableId="Saturday"
        ignoreContainerClipping={false}
        isDropDisabled={false}
        type="DEFAULT"
      >
        <Component />
      </Connect(Droppable)>
      <Connect(Droppable)
        direction="vertical"
        droppableId="Sunday"
        ignoreContainerClipping={false}
        isDropDisabled={false}
        type="DEFAULT"
      >
        <Component />
      </Connect(Droppable)>
    </div>
  </DragDropContext>
</Fragment>
`;
